pkg/meta/model/index.go

```diff
@@ -172,6 +173,260 @@ type FullTextIndexInfo struct {
 	// TODO: Add other options
 }
 
+// HybridIndexInfo is the information of HYBRID index of a column.
+type HybridIndexInfo struct {
+	FullText []*HybridFullTextSpec `json:"fulltext,omitempty"`
+	Vector   []*HybridVectorSpec   `json:"vector,omitempty"`
+	Inverted []*HybridInvertedSpec `json:"inverted,omitempty"`
+	Sort     *HybridSortSpec       `json:"sort,omitempty"`
+}
```

Comment:
I'm thinking of the way that we add a boolean field `OnlyFullTextCols` to represent whether this index is created by `create fulltext index ...`.
Then we will only need one IndexInfo for two types of tici indexes.

The `model.IndexInfo` will only need one `HybridIndexInfo` field. Don't need to store the `FullTextIndexInfo` anymore.